cdc/owner/changefeed.go

```diff
@@ -202,7 +203,15 @@ LOOP:
 		failpoint.Return(errors.New("failpoint injected retriable error"))
 	})
 	if c.state.Info.Config.CheckGCSafePoint {
-		err := util.CheckSafetyOfStartTs(ctx, ctx.GlobalVars().PDClient, c.state.ID, checkpointTs)
+		// Ensure TiDB GC safepoint does not exceed the checkpoint in the next
+		// 10 mintues.
+		//
+		// It is necessary to update TTL, because the service GC safepoint is
+		// set to 1 hour TTL during creating. We need to shrink the TTL to
+		// unblock TiDB GC.
+		ensureTTL := int64(10 * 60)
+		err := gcutil.EnsureChangefeedStartTsSafety(
+			ctx, ctx.GlobalVars().PDClient, c.state.ID, ensureTTL, checkpointTs)
```

Comment:
In fact, this gc safepoint is useless here. 
Should we set the gcttl smaller(eg. 10s?) and add a comment: We check again and delete the service GC safepoint here.